portal-web/test/functional/com/liferay/portalweb/paths/plugin/workflow/WorkflowMetrics.path

```diff
@@ -209,6 +234,11 @@
 	<td>xpath=(//span[contains(text(),'${key_assignee}')]/ancestor::td[contains(@data-testid,'assigneeProfileInfo')]/following-sibling::td//span)[1]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>METRICS_PERFORMANCE_BY_ASSIGNEE_TABLE_POSITION</td>
+	<td>xpath=(//div[5]//div[1]//div[2]//div[1]//table[1]//tbody[1]//tr[${key_rowNumber}]//td[1])</td>
```

Comment:
Hi @Adriverforlife ,
I was wondering about this locator and this is not much readable... You are using this locator in order to check the number of rows of the Performance by Assignee table, right?

Do you think that the following locator can work?
_(//td[contains(@class, 'assignee-name')])[${key_rowNumber}]_

It would become more readable and easier to edit in case the locator is changed. What do you think?

Reply:
That works. I can change that over.